yuuna/lib/src/pages/implementations/subtitle_options_dialog_page.dart

```diff
@@ -459,4 +567,53 @@ class _SubtitleOptionsDialogPage
   void executeSet() async {
     await setValues(saveOptions: false);
   }
+
+  /// Pick a font file with a built-in file picker.
+  Future<bool> pickFontFile() async {
+    FilePickerResult? result = await FilePicker.platform.pickFiles(
+      type: FileType.custom,
+      allowedExtensions: ['ttf', 'otf'],
+    );
+    if (result != null) {
+      _fontNameController.text = result.files.single.name.split('.').first;
+      var custom = FontLoader(_fontNameController.text);
+      custom.addFont(loadFont(result.files.single.path ?? ''));
+      await custom.load();
+      return true;
+    }
+    return false;
+  }
+
+  Future<ByteData> loadFont(String path) async {
+    File file = File(path);
+    Uint8List bytes = await file.readAsBytes();
+    return ByteData.view(bytes.buffer);
+  }
+
+  void showColorPicker(String target) {
+    showDialog(
+        context: context,
+        builder: (context) {
+          return AlertDialog(
+            title: const Text('Pick color'),
```

Comment:
Hardcoded string, I would remove the title here altogether, since it's pretty obvious from the content that the activity is color picking. I also noticed that this doesn't have buttons to cancel or confirm the color selection before changing the color.

Does this also change the hex value in text on the `TextEditingController` you've made for `fontColor` and `outlineColor`, so the user can confirm that they are changing the setting before saving all the settings?